packages/react-data-grid/src/ColumnComparer.ts

```diff
@@ -0,0 +1,24 @@
+import { isElement } from 'react-is';
+
+export function sameColumn<A, B>(a: A, b: B): boolean {
```

Comment:
We should probably replace the generic types once we know how this function is used.